README.md

```diff
@@ -3,32 +3,133 @@ consul-java-client
 
 Consul client for our Java services.
 
+* minimized bootstrapping amount
+* easy Spring Boot and Spring support
+* only one health check is supported
```

Comment:
I would have written something like, because of the underlying client ... Because it is indeed a limitation.